portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -713,6 +713,16 @@
 	<td>//button[contains(@type,'submit') and contains(.,'${key_tab}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>TRY_REDIRECTION_BTN</td>
```

Comment:
Please rename this and the ones above and below. Don't use shorthand (i.e. "btn") in paths. And "button" is redundant since the name is already in the file. "TRY_REDIRECTION" is enough.